<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtCMBProfileWedgeFunction</class>
 <widget class="QWidget" name="qtCMBProfileWedgeFunction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Slope</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="symmetric">
     <property name="text">
      <string>Symmetric</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Left: 1 /</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="LeftSlope">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Right: 1/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="RightSlope">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="4">
    <widget class="QCheckBox" name="relative">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Relative</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Displacement</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QGroupBox" name="weightingBox">
     <property name="title">
      <string>Weighting Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6" rowstretch="0,0,0,0" columnstretch="0,1,0,0">
      <property name="margin">
       <number>5</number>
      </property>
      <item row="2" column="3">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>1                     </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>129</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QGroupBox" name="WeightingSplineControl">
        <property name="title">
         <string>Spline</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <property name="margin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QFrame" name="weightingChartFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Base Width</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="baseWidth">
     <property name="decimals">
      <number>5</number>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="depth">
     <property name="decimals">
      <number>5</number>
     </property>
     <property name="minimum">
      <double>-1000000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QCheckBox" name="clamp">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Clamp</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="Dig">
     <property name="text">
      <string>Dig</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
